tests/test_runner.py

```diff
@@ -36,7 +41,142 @@
         ),
     ],
 )
-def test(entrypoint, kwargs_list, expected_output):
+def test(entrypoint, kwargs_list, expected_output, global_config_fixture, environ_annotations_fixture):
+    # GIVEN
+    global_config_fixture.experimental_features["script_annotations"] = True
+    global_config_fixture.experimental_features["script_runner"] = True
+
+    # WHEN
+    output = _runner(entrypoint, kwargs_list)
+    # THEN
+    assert serialize(output) == expected_output
+
+
+@pytest.mark.parametrize(
+    "entrypoint,kwargs_list,expected_files",
+    [
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_param",
+            [{"name": "a_number", "value": "3"}],
+            [{"path": "hera/outputs/parameters/successor", "value": "4"}],
+        ),
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_artifact",
+            [{"name": "a_number", "value": "3"}],
+            [{"path": "hera/outputs/artifacts/successor", "value": "4"}],
+        ),
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_artifact_path",
+            [{"name": "a_number", "value": "3"}],
+            [{"path": "file.txt", "value": "4"}],
+        ),
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_artifact_and_param",
+            [{"name": "a_number", "value": "3"}],
+            [
+                {"path": "hera/outputs/parameters/successor", "value": "4"},
+                {"path": "hera/outputs/artifacts/successor", "value": "5"},
+            ],
+        ),
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_two_params",
+            [{"name": "a_number", "value": "3"}],
+            [
+                {"path": "hera/outputs/parameters/successor", "value": "4"},
+                {"path": "hera/outputs/parameters/successor2", "value": "5"},
+            ],
+        ),
+        (
+            "tests.script_annotations_outputs.script_annotations_output:script_two_artifacts",
+            [{"name": "a_number", "value": "3"}],
+            [
+                {"path": "hera/outputs/artifacts/successor", "value": "4"},
+                {"path": "hera/outputs/artifacts/successor2", "value": "5"},
+            ],
+        ),
+    ],
+)
+# @patch("hera.workflows.runner.importlib")
```

Comment:
Do we not need to mock the importlib?

Reply:
Not anymore, since we're not using different `tmp_path`s for different tests